api/capi/src/nnstreamer-capi-pipeline.c

```diff
@@ -71,6 +71,7 @@ construct_element (GstElement * e, ml_pipeline * p, const char *name,
     ml_pipeline_element_e t)
 {
   ml_pipeline_element *ret = g_new0 (ml_pipeline_element, 1);
+  g_assert (ret != NULL);
```

Comment:
Asserting (killing the app) in a API library may be... a bit too much.

Can we let it return error along with the users of ``construct_element```?